more/pixels/pixelAssay.js

```diff
@@ -0,0 +1,17 @@
+import harden from '@agoric/harden';
+
+import { makePixelStrategy } from './pixelStrategy';
+import { makeAssayMaker } from '../../core/assay';
+
+// A pixelList is a naive collection of pixels in the form:
```

Comment:
I created an issue to make a generic List Assay. (https://github.com/Agoric/ERTP/issues/101) I think it will be generally useful.